src/main/resources/db/changelog/changelog-master.xml

```diff
@@ -7,5 +7,6 @@
   <include file="changes/v1.0.0/changelog-v1.0.0.xml" relativeToChangelogFile="true"/>
   <include file="changes/v1.0.1/changelog-v1.0.1.xml" relativeToChangelogFile="true"/>
   <include file="changes/v1.0.2/changelog-v1.0.2.xml" relativeToChangelogFile="true"/>
+  <include file="changes/v1.0.3/changelog-v1.0.3.xml" relativeToChangelogFile="true"/>
```

Comment:
I'd bump this to 1.1.0, to try and keep it reasonably in sync with the project version